specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json

```diff
@@ -313,7 +313,7 @@
     "CompressionLevel": {
       "title": "Microsoft.Azure.Management.DataFactories.Models.CompressionLevel",
       "description": "All available compression levels.",
-      "type": "string",
```

Comment:
All these changes in stable API will be breaking.

Reply:
They are not breaking - the type change is from string to object. We had this kind of change in the past and SDK teams signed off on this.

Reply:
It might be a mistake that sign off was received earlier.. . These are breaking changes for clients . Added breaking changes tag.

---
In reply to: [286142468](https://github.com/Azure/azure-rest-api-specs/pull/5967#discussion_r286142468) [](ancestors = 286142468)